homeassistant/components/recorder/statistics.py

```diff
@@ -1894,68 +1894,79 @@ def get_latest_short_term_statistics(
     metadata: dict[str, tuple[int, StatisticMetaData]] | None = None,
 ) -> dict[str, list[StatisticsRow]]:
     """Return the latest short term statistics for a list of statistic_ids."""
+    _LOGGER.warning(
+        "Calling get_latest_short_term_statistics is deprecated; "
+        "use get_latest_short_term_statistics_with_session instead"
+    )
```

Comment:
This code is never called anymore in the codebase but I left it in for custom components

Reply:
This change looks bigger than it actually is. Its mostly whitespace

Reply:
I decided to take it out since its never used in our code besides to compile stats and now replaced with `get_latest_short_term_statistics_with_session` to match all the other functions